src/main/java/info/tehnut/soulshards/core/ConfigSoulShards.java

```diff
@@ -1,171 +0,0 @@
-package info.tehnut.soulshards.core;
-
-import com.google.common.collect.Maps;
-import com.google.common.collect.Sets;
-import info.tehnut.soulshards.core.data.MultiblockPattern;
-import net.minecraft.entity.EntityCategory;
-import net.minecraft.entity.LivingEntity;
-import net.minecraft.util.Identifier;
-import net.minecraft.util.registry.Registry;
-
-import java.util.Map;
-import java.util.Set;
-
-public class ConfigSoulShards {
```

Comment:
I don't particularly like this class split

Reply:
Reverted.